common/interpreter/wawaka_wasm/CMakeLists.txt

```diff
@@ -23,7 +23,10 @@ IF (NOT EXISTS "$ENV{WASM_SRC}")
   MESSAGE(FATAL_ERROR "WASM_SRC environment variable not defined!")
 ENDIF()
 
+# TODO: check if WASM_MODE has been defined
```

Comment:
Can't you just implement the TODO with the blueprint from WASM_SRC a few lines above?

Reply:
Yes, so I had originally already added the check but didn't want to force everyone to add the new environment variable right away. I can add it back in.